routes/students.js

```diff
@@ -47,6 +47,50 @@ router.get('/teacher/:teacherId', async (req, res) => {
   }
 });
 
+// get all students groups and students for a given site
+router.get('/site/:siteId', async (req, res) => {
+  try {
+    const { siteId } = req.params;
+    isNumeric(siteId, 'Site Id must be a Number');
+    const students = await pool.query(
+      `SELECT student_group.*, relation.students
```

Comment:
I don't think we need all the info about student groups - I think you can just make it like the one below (only returns a list of students)